types/src/version.rs

```diff
@@ -42,30 +47,42 @@ const MAINNET_SCHEDULE: [Upgrade; 5] = [
         height: UPGRADE_TAPE_HEIGHT,
         network: NetworkVersion::V5,
     },
+    Upgrade {
+        height: UPGRADE_KUMQUAT_HEIGHT,
+        network: NetworkVersion::V6,
+    },
 ];
 
 /// Specifies the network version
-#[derive(Debug, PartialEq, Clone, Copy, PartialOrd)]
+#[derive(Debug, PartialEq, Clone, Copy, PartialOrd, Serialize)]
```

Comment:
Should this be serialize repr? what is this needed for?

Reply:
looks like this should indeed be serialize_repr